t/14-lb_req_dc_rr.t

```diff
@@ -171,9 +171,9 @@ local_dc must be a string
             local peers = {
                 {host = '10.0.0.1', data_center = 'dc2'},
 
-                {host = '127.0.0.1', data_center = 'dc1'},
+                {host = '127.0.0.1'},
                 {host = '127.0.0.2', data_center = 'dc1'},
-                {host = '127.0.0.3'}
+                {host = '127.0.0.3', data_center = 'dc1'}
```

Comment:
The patch would be more concise and less disruptive if we keep this test the way it is, only change the expected results.

Reply:
I changed this part because the bug is triggered when the first host has no `data_center` field.